pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -171,6 +197,15 @@ func (sna *SnapshotAlpha) CreateFromSource(ctx context.Context, source *Source,
 	return sna.WaitOnReadyToUse(ctx, snapshotName, namespace)
 }
 
+// CreateContentFromSource will create a 'VolumesnaphotContent' for the underlying snapshot source.
```

Comment:
New API - `CreateContentFromSource`. Allows caller to create just a `VolumeSnapshotContent` if needed.